### PR TITLE
Add Wartortle pokemon

### PR DESCRIPTION
**Pull request by miguelramirezr**

### CODE REVIEW

**Comment by miguelramirezr** [comment]
@mhiliri Can you review this pull request?

File: index.html [hunk 1]
```diff
@@ -28,6 +28,10 @@ <h1>Pokedex</h1>
     </tr>
     <tr>
       <td><center>2</center></td>
+      <td><img src="images/008Wartortle.png"></td>
```
Comment (mhiliri) [inline index.html]: Please add the correct number for Tortle and fix the merge conflict

Comment (mhiliri) [reply index.html]: @miguelramirezr  please fix the merge conflict

**Comment by miguelramirezr** [comment]
Conflicts solved.

**Review by mhiliri — APPROVED** [review]

**Pull request merged**